app/Http/Controllers/Rest/UsersController.php

```diff
@@ -72,7 +73,7 @@ public function count(Request $request)
 
     public function browse(Request $request)
     {
-        return self::json([], 501, ['message' => 'not yet implemented <3']);
+//        return self::json([], 501, ['message' => 'not yet implemented <3']);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)